packages/core/src/tools/internal/inline-tools/Bold/index.ts

```diff
@@ -0,0 +1,80 @@
+import type { InlineTool } from '../../../../entities/InlineTool.js';
+import type { InlineFragment, TextRange } from '@editorjs/model';
+import { FormattingAction } from '@editorjs/model';
+import { createInlineToolName, IntersectType, type InlineToolName } from '@editorjs/model';
+import { make } from '@editorjs/dom';
+
+/**
+ * Bold Tool
+ *
+ * Inline Toolbar Tool
+ *
+ * Makes selected text bolder
+ */
+export default class BoldInlineTool implements InlineTool {
+  /**
+   * Specifies Tool as Inline Toolbar Tool
+   * @returns {boolean}
+   */
+  public static isInline = true;
+
+  public name: InlineToolName = createInlineToolName('Bold');
+
+  public intersectType: IntersectType = IntersectType.Extend;
+
+  /**
+   * Renders wrapper for tool without actual content
+   * @returns Created html element
+   */
+  public create(): HTMLElement {
+    return make('b');
+  }
+
+  /**
+   * Returns formatting action and range for it to be applied
+   * @param index - index of current text selection
+   * @param fragments - all fragments of the bold inline tool inside of the current input
+   */
+  public getAction(index: TextRange, fragments: InlineFragment[]): {
+    /**
+     * Formatting action - format or unformat
+     */
+    action: FormattingAction;
+
+    /**
+     * Range to apply formatting action
+     */
+    range: TextRange;
+  } {
+    return {
+      action: this.checkState(index, fragments) ? FormattingAction.Unformat : FormattingAction.Format,
+      range: index,
+    };
+  };
+
+  /**
+   * Returns state of the bold inline tool
+   * @param index - index of current selection
+   * @param fragments - all fragments of the bold inline tool inside of the current input
+   * @returns true if tool is active, false otherwise
+   */
+  public checkState(index: TextRange, fragments: InlineFragment[]): boolean {
```

Comment:
should it be public?

Reply:
also, it would be better to name method returning boolean starting with `is...` — `isActive` for example

Reply:
it should be public because toolbar would need to know state of each tool